### ISSUE TITLE
documentation for install on windows.

### ISSUE THREAD

**Issue by dbfreem** [ethereum/web3.py#2292]
### What was wrong?

I have noticed that many people ask about how to install web3.py on windows. I was thinking about updating the docs with the preferred way of installing web3.py on windows. I have installed web3.py on windows using WSL but not sure if that would be the official way that Ethereum wants to support windows install. 

I am happy to update the docs but wanted to open this issue for discussion. 

One example is #2290

**Comment by kclowes** [comment]
None of the maintainers have windows machines, so it would be great to get some more documentation out there! I am not familiar enough with WSL to know if we should make it the "official" way or not, but I definitely think we could add a note in the Quickstart or in the Troubleshooting section? /cc @marcgarreau

**Comment by dbfreem** [comment]
I did a little more research on this. Looks like the [bitarray repo](https://github.com/ilanschnell/bitarray#installation)  suggest using conda or going to [Chris Gohlke](https://www.lfd.uci.edu/~gohlke/pythonlibs/#bitarray) if you want precompiled binaries. The problem with the precompiled wheels by Chis is that it is my understanding that they are not official wheels or supported. 

The second issue if someone wanted to the wheels precompiled by Chris is that there isn't one that meets the current bitarray requirment of eth-account. bitarray < 1.3.0 >= 1.2.1

I don't have any experience with conda so not sure if that is a viable option.

This also may need to be an issue on the eth-account repo and not web3.py

Thought???

**Comment by kclowes** [comment]
Yeah, we have an issue on eth-account to look into removing the bitarray dependency entirely. I think that's probably the "right" way to go. In the meantime, I think it's probably still worth adding something to at least the Troubleshooting docs in the web3.py repo that suggests using WSL or conda, assuming we can verify that conda works. I also don't use conda at all. Thanks for looking into it @dbfreem!

**Comment by fselmo** [comment]
@dbfreem, thanks for this. If you have a Windows machine and are willing to give the above a shot and document it, I think it can be quite useful for Windows users until the dependency issue gets resolved. I can keep this open for now. Is that of interest to you at all?

None of the maintainers have Windows machines so it will be hard for us to come up with a good solution here.

**Comment by dbfreem** [comment]
@fselmo yea I plan to pick this up next to work on the docs. I don't have conda but I can put something together for WSL.
